### PR TITLE
Hide the invite member button for users that don't have access to a team or insufficient privileges

### PR DESCRIPTION
**Pull request by cstns**
## Description

Hides the Invite member button for users that don't have sufficient privileges or insufficient privileges

## Related Issue(s)

https://github.com/FlowFuse/flowfuse/issues/4284

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: frontend/src/components/PageHeader.vue [hunk 1]
```diff
@@ -117,7 +118,7 @@ export default {
             ].filter(option => option !== undefined)
         },
         showInviteButton () {
-            return this.$route.name !== 'TeamMembers'
+            return this.team && this.teamMembership >= 30 && this.$route.name !== 'TeamMembers'
```
[the remark this answers is not in the document]
Comment (cstns) [reply frontend/src/components/PageHeader.vue:121]: Good observation! I initially wanted to use the permissions mixin but couldn't find the appropiate permission!

Will update shortly.

[the remark this answers is not in the document]
Comment (cstns) [reply frontend/src/components/PageHeader.vue]: Done!

**Review by Steve-Mcl — APPROVED** [review]
verified on pre-staging (logged in as owner, member and viewer & verified correct visibility)

**Comment by Steve-Mcl** [comment]
merge when tests go green.

**Comment by cstns** [comment]
the ui failing test is the  devices/bulk.spec.js. Will merge if ok with you, @Steve-Mcl!

**Comment by Steve-Mcl** [comment]
> the ui failing test is the devices/bulk.spec.js. Will merge if ok with you, @Steve-Mcl!

Yes please, then re-run these ui tests.

**Pull request merged**